app/assets/stylesheets/content/_in_place_editing.sass

```diff
@@ -30,6 +30,17 @@ $inplace-edit--border-color: #ddd
 $inplace-edit--dark-background: #f8f8f8
 $inplace-edit--color--very-dark: #cacaca
 
+
+%inline-datepicker-container
+  display: none
+  z-index: 99999
+  right: 0
+  position: absolute
+
+%span-margin
```

Comment:
@hirtie-maxim placeholder selectors like this are global, not within the context of this partial. Please give this placeholder selector a less generic name, e.g. `%inplace-edit--span-margin`.